src/components/App/components/UserInput/UserInput.js

```diff
@@ -71,7 +71,6 @@ function UserInput(props) {
                   id: 'font-size',
                   type: 'number',
                   min: MIN_FONT_SIZE_IN_PX,
-                  max: MAX_FONT_SIZE_IN_PX,
```

Comment:
Sure!

First, note that there is a difference between constraining the font size input and the Preview.

In the font size input, we are constraining a number, regardless of the font size unit. For example, if the max is 36, it means that user can get to 36px or 36pt, but not 37px or 37pt.

In the Preview, we are constraining using a unit. For example, if the max is 36px, it means that we never show text that is 37px. Even if user puts 36pt, which is 48px, we'll cut the font size down to the max of 36px.

By default, I'd say let's not constrain either.
The problem with this is, that users could get to 3 digit font sizes which we don't handle well (input is too narrow). So, it doesn't look good, and also I think it's not very useful. This is why I suggest we do constrain the user input.

Constraining user input means that the Preview will be constrained as well. We don't need to write code to constrain it! We just need to note that, if font size input is constrained by 36, for example, then the Preview will be automatically constrained by 48px (so the max number is not quite the same in both).

If this makes sense to you, I suggest we bring back `MAX_FONT_SIZE` and add a comment:
```js
export const MAX_FONT_SIZE = 21; // Maximum font size input value
```
(and let's add a similar comment for `MIN_FONT_SIZE`)

I think adding `IN_PX` only caused a lot of confusion.